sql/src/main/java/io/crate/planner/projection/WriterProjection.java

```diff
@@ -60,6 +60,9 @@
     private boolean isDirectoryUri;
```

Comment:
could you pls add a WriterPojectionTest (unit test) to test the streaming of this class?